.gitlab-ci.yml

```diff
@@ -153,14 +153,17 @@ test-on-netgear-rax40:
   variables:
     # TESTS_TO_RUN needs to be set by the user (or the pipeline schedule)
     GIT_CLONE_PATH: "/builds/prpl-foundation/prplMesh/"
+    DEVICE_UNDER_TEST: prplmesh
```

Comment:
Maybe add a comment about this variable as well.

Reply:
Done.